packages/utilities/src/utilities/altlangs/altlangs.js

```diff
@@ -0,0 +1,29 @@
+import root from 'window-or-global';
+
+/**
+ * Utility to grab all alternative languages on the page. this scrapes the page
+ * of all <link rel="alternate" hreflang="" href="" /> elements and returns
+ * as a readable object
+ *
+ * @example
+ * import { altlangs } from '@carbon/ibmdotcom-utilities';
+ *
+ * const langs = altlangs();
+ *
+ * console.log(langs); // { 'us-en': 'https://www.ibm.com/us-en', ... }
+ *
+ * @returns {object} object of available languages and corresponding URLs
+ *
+ */
+function altlangs() {
+  const links = root.document.querySelectorAll('link[rel="alternate"]');
+  const langs = {};
+
+  links.forEach(link => {
+    langs[link.getAttribute('hreflang')] = link.getAttribute('href');
```

Comment:
Is there any value in being explicit with lang and country, and pulling those out for easy access?

Something like this...
```javascript
const hreflang = link.getAttribute('hreflang');
const splitHrefLang = hreflang.split('-');

langs[hreflang] = link.getAttribute('href');
langs['cc'] = splitHrefLang[0];
langs['ll'] = splitHrefLang[1];
```

Reply:
Good suggestion. I'll add that!

Reply:
Was thinking a bit on this, the list of available country/language would essentially be cross referenced with the country list here: https://www.ibm.com/common/v18/js/data/jsononly/locale/usen-locale.json

This list already has it in the format of `cc-lc`, so my hunch is that when generating the available list, it would check to see if `altlangs` has those sets of keys available in that object. We might be ok without breaking it out?

cc: @annawen1 @kennylam @andysherman2121 @sannminwin

Reply:
That's probably fair.  I'm cool to merge this if y'all are. We can always split them out down the road if the need arises.

Reply:
hmmm the `LocaleAPI.verifyLocale()` method expects the `cc` and `lc` as separate parameters, so if we want to reuse that I think we should break it out.
```
static verifyLocale(cc, lc, list) { ... }
```

Reply:
Thanks @annawen1 and @photodow , I'll merge in as is then!